packages/fxa-auth-db-mysql/lib/db/schema/patch-104-105.sql

```diff
@@ -0,0 +1,58 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('104');
+
+CREATE PROCEDURE `resetAccount_14` (
```

Comment:
Is it deliberate that you didn't update the call in `lib/db/mysql.js`?

Reply:
Nvm, just noticed you didn't actually ask for review yet, sorry!

Reply:
it is not; thanks!